package.json

```diff
@@ -8,7 +8,7 @@
     "async-eventemitter": "^0.2.2",
     "ethereum-common": "0.1.0",
     "ethereumjs-account": "^2.0.3",
-    "ethereumjs-block": "^1.2.2",
+    "ethereumjs-block": "git+https://github.com/holgerd77/ethereumjs-block.git#byzantium-changes",
```

Comment:
Isn't this merged?

Reply:
No, that's work in progress, the PR you merged was just a prerequisite for this to be implemented. Finished EIP 100, will tomorrow have a look at EIP 649.

Reply:
@axic Also, before merging the Metro-Byzantium changes into ``ethereumjs-block``, I would like to have this PR https://github.com/ethereumjs/ethereumjs-vm/pull/189 to be merged, otherwise new installations target the current network will break (at lease when there is a npm ``ethereumjs-block`` release with the changes).